test/helper.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+/**
+ * Removes the line break in the given file contents.
+ */
+exports.removeLineBreak = function (contents) {
+    return contents.toString().replace(/\r?\n|\r/g, '');
+};
```

Comment:
I mean, why do you wan't to remove the line break? `gulp-ejs` should not change the behaviour of `ejs` in anyway, don't you agree? Also that would be a breaking change for everyone using the plugin.